frontend/packages/core/src/Input/text-field.tsx

```diff
@@ -1,37 +1,122 @@
-import React from "react";
-import type { InputLabelProps, TextFieldProps as MuiTextFieldProps } from "@material-ui/core";
+import * as React from "react";
+import styled from "@emotion/styled";
+import type {
+  InputProps as MuiInputProps,
+  StandardTextFieldProps as MuiStandardTextFieldProps,
+} from "@material-ui/core";
 import { TextField as MuiTextField } from "@material-ui/core";
-import styled from "styled-components";
+import ErrorIcon from "@material-ui/icons/Error";
+import _ from "lodash";
 
 const KEY_ENTER = 13;
-const StyledTextField = styled(MuiTextField)`
-  ${({ theme, ...props }) => `
-  display: flex;
-  width: 100%;
-  max-width: ${props["data-max-width"] || "500px"};
-  margin: 15px;
-  .MuiInputLabel-root {
-    color: ${theme.palette.text.primary};
-  }
-  .MuiInput-underline:after {
-    border-bottom: 2px solid ${theme.palette.accent.main};
-  }
-  `}
-`;
 
-export interface TextFieldProps {
-  maxWidth?: string;
+const BaseTextField = ({ InputProps, InputLabelProps, ...props }: MuiStandardTextFieldProps) => (
+  <MuiTextField
+    InputLabelProps={{ ...InputLabelProps, shrink: true }}
+    InputProps={{ ...InputProps, disableUnderline: true }}
+    fullWidth
+    {...props}
+  />
+);
+
+const StyledTextField = styled(BaseTextField)({
+  ".MuiInputLabel-root": {
+    fontSize: "13px",
+    fontWeight: "bold",
+    transform: "scale(1)",
+  },
+
+  ".MuiInputLabel-root, .MuiInputLabel-root.Mui-focused": {
+    color: "rgba(13, 16, 48, 0.6)",
+  },
+
+  ".MuiInputLabel-root.Mui-disabled": {
+    color: "rgba(13, 16, 48, 0.38)",
+  },
+
+  ".MuiInputLabel-root.Mui-error": {
+    color: "#db3615",
+  },
+
+  ".MuiInputBase-root": {
+    border: "1px solid rgba(13, 16, 48, 0.38)",
+    borderRadius: "4px",
+    fontSize: "16px",
+    padding: "14px 16px",
```

Comment:
I believe this padding is 13px on top and bottom in the designs

Reply:
figma has line height, for multiline content which is usually dense (e.g. JSON) didn't want line height, this padding is a pixel match with figma